examples/data_preprocessing/police_killing/data_preprocessing_with_regex.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
many corrections from `data_preprocessing_with_keywords.ipynb` file are also relevant here (e.g. all V-ing -> V, dict/list types -> Dict/List from the `typing` library, upper case in section naming, some descriptions, final words etc)